Update agent readiness score check docs (#5393)
mintlify/docs: cli/commands.mdx

| `robotsTxtAllowsAI` | The `robots.txt` file does not block AI crawlers. |
| `sitemapExists` | There is a sitemap available for page discovery. |
| `structuredData` | The homepage contains [JSON-LD](https://json-ld.org/) structured data (`<script type="application/ld+json">`). Reports the number of JSON-LD blocks and the schema types found. |
| `responseLatency` | The site responds within an acceptable time for agents. |

> Some checks are children of other checks. When a parent check fails, its children are automatically marked as failed. For example, `llmsTxtValid` only passes if `llmsTxtExists` passes first. The overall score uses weighted scoring, so higher-impact checks contribute more to your score.

Reviewer: ```suggestion
Some checks only run if a check they depend on passes. If a check fails, none of the checks that depend on it run. They automatically fail. For example, `llmsTxtValid` only passes if `llmsTxtExists` passes first.

The overall score uses weighted scoring, so higher-impact checks contribute more to your score.
```
